templates/icinga2/services.conf.j2

```diff
@@ -303,10 +303,30 @@ apply Service "ups load" {
   vars.datgroup = "ups"
 }
 
+apply Service "systemd" for (systemd => config in host.vars.systemd)  {
+  import "normal-service"
+  check_command = "systemd"
+  vars.notify = false
```

Comment:
Never set `notify` to false in the service definition. This is only done for the apt check to avoid one alert per machine as soon as an update is published and needs strong motivation.